src/api/lib/search/search.ts

```diff
@@ -32,7 +31,9 @@ export class SearchService extends Service {
       this.request<IADSApiSearchResponse>(config).then(
         (result) => {
           result.match(
-            ({ response }) => resolve(ok(response)),
+            ({ response, stats }) => {
+              stats ? resolve(ok({ response, stats })) : resolve(ok({ response }));
+            },
```

Comment:
I'm not a big fan of this, since we have an extra property to deal with even though like 90% of the queries won't have stats, but I don't think we should create a different request or anything for this.
```suggestion
            ({ response, stats }) => resolve(ok({ response, stats }));
```
Should just always return `stats` now even if it's `undefined` most of the time, so it's consistent

Reply:
Done.